languages/tree-sitter-stack-graphs-typescript/rust/tsconfig.rs

```diff
@@ -38,12 +34,8 @@ impl FileAnalyzer for TsConfigAnalyzer {
         _cancellation_flag: &dyn tree_sitter_stack_graphs::CancellationFlag,
     ) -> Result<(), tree_sitter_stack_graphs::LoadError> {
         // read globals
-        let pkg_name = globals
-            .get("PACKAGE_NAME")
-            .map(String::as_str)
-            .unwrap_or("");
         let proj_name = globals
-            .get("PROJECT_NAME")
+            .get(crate::PROJECT_NAME_VAR)
             .map(String::as_str)
             .unwrap_or("");
```

Comment:
Should we blow up here if the project name isn’t set, or is there an advantage to continuing on with `””`?

Reply:
See my earlier reply on this.